package.json

```diff
@@ -53,21 +53,23 @@
     "@babel/runtime": "^7.23.9",
     "@stoplight/spectral-core": "^1.8.1",
     "@stoplight/spectral-rulesets": "^1.18.1",
-    "@useoptic/optic": "^0.43.0",
-    "@useoptic/json-pointer-helpers": "^0.53.16",
-    "@useoptic/openapi-io": "^0.53.16",
-    "@useoptic/openapi-utilities": "^0.53.11",
-    "@useoptic/rulesets-base": "^0.53.16",
+    "@useoptic/optic": "^0.54.4",
+    "@useoptic/json-pointer-helpers": "^0.54.4",
+    "@useoptic/openapi-io": "^0.54.4",
+    "@useoptic/openapi-utilities": "^0.54.4",
+    "@useoptic/rulesets-base": "^0.54.4",
     "chalk": "^4.0.0",
     "change-case": "^4.1.2",
-    "commander": "10.0.0",
+    "commander": "11.1.0",
     "find-parent-dir": "^0.3.1",
     "fs-extra": "^10.0.0",
     "yaml": "^2.3.4",
     "yargs": "~17.6.0"
   },
   "overrides": {
-    "postman-collection": "4.2.0"
+    "postman-collection": "4.2.0",
+    "pac-resolver": "7.0.1",
+    "socks": "2.7.3"
```

Comment:
Don't need these overrides, the packages that have these listed as dependencies do not point to exact versions. Since these versions have already have entries in our package lock there is no need to explicitly declare overrides, and having them could hurt dependency resolution in the future